running-your-first-ot-server/setup-webserver-linux.md

```diff
@@ -0,0 +1,201 @@
+# Getting started
+Rent a Ubuntu Server 18.04 or higher from a trusted hosting company such or install for the sake of learning experience Ubuntu Server 18.04 or higher in VirtualBox.
```

Comment:
Technically, WSL could be used on Windows as well instead of VirtualBox.

Reply:
Using WSL as server would cause many people with a lot of issues. I been through issues with it as well. Giving more options would give people a better feeling for themselves to pick the way they find to use the best. VirtualBox is given as example since its great for new people. When you've made a mistake you delete the VM and create a new one and start over. We are making guides for people which has no knowledge.

Reply:
VirtualBox also empowers them with advanced solutions, such as using USB passthrough to a separate Wifi dongle. This ensures Windows is not interfering, in any manner whatsoever, with the TFS servers internet connectivity. An important elimination of complexity in the typical diagnosing self-hosted login issues thread in the support forum. Though admittedly outside the scope of this particular guide.

Still, if both options are presented, VirtualBox should be preferred, because it presents the full native Linux environment. And if there is an advanced guide on the matter they follow later, it would definitely introduce them to VBox's seamless mode, which is a game changer for really experiencing the power of a Linux desktop environment without leaving their comfortable safety of Windows. Call it optimism for the best result.